scripts/test

```diff
@@ -63,9 +63,11 @@ compose() {
 
   # Guarantee clean environment
   compose down -v
+  docker-compose -f docker-compose.migrate.yaml -p "${service}-test" down -v
 
   # Ensure container images are up to date
   compose build
+  docker-compose -f docker-compose.migrate.yaml -p "${service}-test" run --rm database-up
 
   # Run tests
   if [ "${debug}" = "true" ]; then
```

Comment:
Do you need a repeat of line 66 command at the end of the file to clean up the service-test container?